### PR TITLE
Hero buttons responsive bug fixed and repo-card also

### PR DESCRIPTION
**Pull request by SharizHussain**
Your **recent button issue** fixed 
and previous **responsive issue** also solved 

please label me approved and merge after check it

### CODE REVIEW

File: package-lock.json
Comment (max-programming) [inline package-lock.json:1]: Can you please delete the lockfile or put it in `.gitignore`?
This repo uses pnpm

File: src/env.mjs
Comment (max-programming) [inline src/env.mjs:1]: I don't think there is a need for this
If it throws an error, you can just remove this env variable from `.env.local` if it is empty

File: package-lock.json
Comment (max-programming) [inline package-lock.json:1]: Can you put the lockfile in gitignore, this project uses pnpm so I don't want to have conflicts with npm

**Review by max-programming — CHANGES_REQUESTED** [review]

**Comment by max-programming** [comment]
Looks good but why do the buttons stick together?

image

**Comment by SharizHussain** [comment]
I now fixed those in latest push

**Comment by max-programming** [comment]
That looks good! Thanks

**Comment by SharizHussain** [comment]
Sure now I have added the package-lock.json file in gitignore and navbar's sign in button was sticky to hacktoberfest is fixed.


WhatsApp Image 2024-10-02 at 01 35 51


Screenshot from 2024-10-02 01-32-39

**Review by max-programming — APPROVED** [review]

**Comment by max-programming** [comment]
@all-contributors add @SharizHussain for bug, code

**Comment by SharizHussain** [comment]
If any bug or UI related feature you can tell me later

Thanks 👍

**Pull request merged**